piker/watchlists.py

```diff
@@ -0,0 +1,65 @@
+import os
+import json
+from collections import defaultdict
+
+from .log import get_logger
+
+log = get_logger(__name__)
+
+def write_sorted_json(watchlist, path):
+    for key in watchlist:
+        watchlist[key] = sorted(list(set(watchlist[key])))
+    with open(path, 'w') as f:
+        json.dump(watchlist, f, sort_keys=True)
+
+
+def make_config_dir(dir_path):
```

Comment:
Hmm I'm wondering now maybe `ensure_config_dir` is a better name and maybe it should `return dir_path`?